### PR TITLE
Manage quota exceeded during backup

### PR DESCRIPTION
**Pull request by zatteo**
Display quota paywall when quota exceeded during backup

### CODE REVIEW

File: src/photos/ducks/backup/index.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const BackupPage = () => {
             <BackupInfo />
             {!isFlagshipApp() ? <InstallAppAlert /> : null}
             <BackupActions />
-            <BackupDescription />
+            {isMobile ? <BackupDescription /> : null}
```
Comment (cballevre) [inline src/photos/ducks/backup/index.jsx:36]: nit: You can also use `&&` to be less verbose
```
{isMobile && <BackupDescription />}
```

File: src/photos/ducks/backup/components/BackupError.jsx [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
 
+import { QuotaPaywall } from 'cozy-ui/transpiled/react/Paywall'
 import Snackbar from 'cozy-ui/transpiled/react/Snackbar'
 import Alert from 'cozy-ui/transpiled/react/Alert'
 
 import { useBackupActions } from '../hooks/useBackupActions'
 
+const shouldDisplayQuotaPaywall = backupError => {
+  return backupError.statusCode === 413
+}
```
Comment (cballevre) [inline src/photos/ducks/backup/components/BackupError.jsx:11]: nit: Maybe extract into a helper file with other business logic app like `parseBackupError`

**Review by cballevre — APPROVED** [review]
Happy to see new usage of the `QuotaPaywall` 🙂

File: src/photos/ducks/backup/index.jsx (hunk 1 above)
Comment (zatteo) [reply src/photos/ducks/backup/index.jsx:36]: Agree about less verbose but trying this way for the next weeks https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx (still not sure about the value but trying 😄)

**Comment by zatteo** [comment]
> Happy to see new usage of the `QuotaPaywall` 🙂

A pleasure to use!

File: src/photos/ducks/backup/index.jsx (hunk 1 above)
Comment (cballevre) [reply src/photos/ducks/backup/index.jsx:36]: Thanks for the article 🙂 I use this way also now to avoid having `0` or `NaN` popping randomly

**Pull request merged**